polyphemus/lib/commands.rb

```diff
@@ -271,7 +71,7 @@ def execute
           restrict!(patient, delete_metis_files: true)
         elsif should_be_restricted
           restrict!(patient)
-        elsif ! should_be_restricted && patient['restricted']
```

Comment:
Note: I think if we want every patient to run every time, we need to also remove this check on the existing `restricted` status for the patient.